packages/tabs/src/index.js

```diff
@@ -1,9 +1,35 @@
-import React, { cloneElement, useState, useRef, forwardRef } from "react";
+import React, {
+  useState,
+  useRef,
+  useContext,
+  forwardRef,
+  createContext
+} from "react";
 import { node, func, number } from "prop-types";
 import warning from "warning";
 import { wrapEvent, useUpdateEffect, makeId, useForkedRef } from "@reach/utils";
 import { useId } from "@reach/auto-id";
 
+const createNamedContext = name => {
+  const context = createContext();
+  context.displayName = name;
+
+  return context;
+};
```

Comment:
Not sure if this `createNamedContext` is relevant for this repo. It was copied over from [React Router](https://github.com/ReactTraining/react-router/pull/6479/files#r233653953) based on the comment left at implementation:

> this avoids static property assignments by hiding it in a tree-shakeable function call (this was preventing tree shaking for simialr reason as .defaultProps)